pkg/cmd/keepassxctemplatefuncs.go

```diff
@@ -133,7 +133,14 @@ func (c *Config) keepassxcOutput(name string, args []string) ([]byte, error) {
 		panic(errors.New("keepassxc.database not set"))
 	}
 
-	if c.Keepassxc.password == "" {
+	var noPassword bool
+	for _, v := range args {
+		if v == "--no-password" {
+			noPassword = true
+		}
+	}
```

Comment:
Thanks for this PR. Having chezmoi parse the arguments to keepassxc is fragile as it depends on chezmoi parsing arguments exactly the same way as keepassxc does.

Instead, I think it's better to have a `keepassxc.promptPassword` [configuration variable](https://www.chezmoi.io/reference/configuration-file/variables/) which defaults to `true` and can be set to `false` to disable the password prompt.

Also, below, `cmd.Stdin` should be set to `os.Stdin` if the user disables the password prompt.

Let me know if you want to update this PR, otherwise I can create one. For adding a configuration variable, commit 8cd18326 gives an idea of what files need to be updated.

Reply:
I think you solution is better as well :+1: 
I'm gonna update the PR soon